Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml

```diff
@@ -21,7 +21,7 @@
   startingGear: HoPGear
   icon: "JobIconHeadOfPersonnel"
   requireAdminNotify: true
-  supervisors: job-supervisors-captain
+  supervisors: job-supervisors-cc # ParkStation - Oligarchy
   canBeAntag: false
   access:
```

Comment:
HoP isn't meant to have AA, they're meant to be a Head-manager role and to communicate among them while fulfilling administrative duties. 
That said, I do think them having almost *no* access as is currently the case may be too restrictive. I think giving them 'soft' access to most departments would be a good middle ground, but access isn't currently set up in such a way that that's easily doable.